tests/transports-governance/test_governance_api_endpoints_comprehensive.py

```
@@ -0,0 +1,1155 @@
+            400,
+        ]  # Might not be implemented
+
+        # Test filtering by model
+        model_stats_response = governance_client.get_usage_stats(model="gpt-4")
+        assert model_stats_response.status_code in [
+            200,
+            400,
+        ]  # Might not be implemented
+
+        # Test date range filtering
+        end_time = datetime.now()
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)